config/examples/paperclip.yml

```diff
@@ -7,7 +7,7 @@ s3: &s3
   storage: 's3'
 
 development:
-  <<: *<%= ENV['PAPERCLIP_STORAGE'] || 'filesystem' %>
+  <<: *<%= ENV['FILE_UPLOAD_STORAGE'] || 'filesystem' %>
 
 test:
-  <<: *<%= ENV['PAPERCLIP_STORAGE'] || 'filesystem' %>
+  <<: *<%= ENV['FILE_UPLOAD_STORAGE'] || 'filesystem' %>
```

Comment:
Forgot to comment here. I'm concerned that this may require an operator change. If so, lets keep using `PAPERCLIP_STORAGE`.

Reply:
I don't think it will break the operator code.
1. Operator already references `FILE_UPLOAD_STORAGE`: https://github.com/3scale/3scale-operator/blob/master/pkg/3scale/amp/component/system.go#L242
2. config files that are for OpenShift deployment already use `FILE_UPLOAD_STORAGE` as well: 
https://github.com/3scale/porta/blob/master/openshift/system/config/paperclip.yml#L13
https://github.com/3scale/porta/blob/master/openshift/system/config/amazon_s3.yml#L16

So this change basically aligns the files for local development with the "on-premises" ones.